types/azure-functions-typed/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json"`

Reply:
As @plantain-00 noted, we would like to keep new packages clean. so no exceptions in the config file.

Reply:
Sorry for the delay, fixed now.